src/nginxconfig/generators/index.js

```diff
@@ -81,6 +82,10 @@ export default (domains, global) => {
         if (domains.some(d => d.python.python.computed))
             files['nginxconfig.io/python_uwsgi.conf'] = toConf(pythonConf(global));
 
+        // Rails
+        if (domains.some(d => d.rails.rails.computed))
+            files['nginxconfig.io/rails.conf'] = toConf(railsConf(domains, global));
```

Comment:
In this file, you're only accessing the first item in `domains` -- this doesn't handle if the user configures multiple domains in the tool with different Rails settings. Should the settings be global instead, or should this file be generated once for each domain configured?